python/pyspark/sql/functions.py

```diff
@@ -341,6 +371,20 @@ def monotonicallyIncreasingId():
     return Column(sc._jvm.functions.monotonicallyIncreasingId())
 
 
+@since(1.5)
+def pi():
+    """
+    Returns the float value that is closer than any other to pi, the ratio of the circumference
+    of a circle to its diameter.
+
+    >>> sqlContext.createDataFrame([(1,)], ['a']).select((pi()).alias('pi')).collect()
```

Comment:
Is it possible to do this more concise? `sqlContext.createDataFrame([(pi(),)], ['a']).collect()` fails: `cannot create an RDD from type: <type 'list'>`. Sorry if the answer is obvious, I am not a python geek.